### PR TITLE
Added some checks and practical commands

### PR DESCRIPTION
**Pull request by macmiranda**

### CODE REVIEW

File: centos7.sh [hunk 1]
```diff
@@ -31,45 +31,60 @@ msg_opt_complete(){
...
   yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
-  yum install -y epel-release
+  # install EPEL using rpm so it works for both RHEL and CentOS
+  yum install -y http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 
   # update base OS
   yum update -y
 
   # install required packages
+  yum install -y http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-3.el7.noarch.rpm
```
Comment (chewitt) [inline centos7.sh:49]: I'm not a fan of pulling explicit versions via direct URLs. IMHO the better way would be to subscribe the extras repo with `subscription-manager repos --enable=rhel-7-server-extras-rpms` and then `yum install container-selinux` and let yum figure out version/mirror to use. We probably need to detect RHEL vs CentOS earlier in the script and then check various repo's are subscribed (indicating the system is licensed) or bail out.

File: centos7.sh [hunk 2]
```diff
@@ -31,45 +31,60 @@ msg_opt_complete(){
...
+  CURL_OPTS="--insecure"
   # install basic packages
   yum install -y yum-utils device-mapper-persistent-data lvm2
 
-  # install docker-ce and EPEL repos
+  # install docker-ce
   yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
-  yum install -y epel-release
+  # install EPEL using rpm so it works for both RHEL and CentOS
+  yum install -y http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```
Comment (chewitt) [inline centos7.sh:43]: Same comment as below .. not a fan of direct installing static rpm files although I accept the version of this one is attempting to be a static target.

File: centos7.sh (hunk 1 above)
Comment (macmiranda) [reply centos7.sh:49]: Unfortunately, the version available in the extra resopitory is too old for docker-ce

File: centos7.sh (hunk 2 above)
Comment (macmiranda) [reply centos7.sh:43]: Me neither, it's just what it takes if you want this to support RHEL

**Pull request merged**